docs(core-features): migrate Android SplashScreen (#1249)
apache/cordova-docs: www/docs/en/dev/core/features/splashscreen/index.md

**Usage Example:**

```xml
<preference name="AndroidWindowSplashScreenAnimatedIcon" value="res/screen/android/splashscreen.xml" />
```

### `AndroidWindowSplashScreenAnimationDuration`

Defines the duration of the icon's animation if an Animated Vector Drwable is supplied as the splash screen image.

**Supported Platforms:**

- Android

**Data Type:** `Number`, in milliseconds

**Default Value:** `undefined`

**Usage Example:**

```xml
<preference name="AndroidWindowSplashScreenAnimationDuration" value="3000"/>
```

### `AndroidWindowSplashScreenBackground`

The splash screen background color.

**Supported Platforms:**

- Android

**Data Type:** `String`, color hex code value.

**Default Value:** `#ffffff`

**Usage Example:**

```xml
<preference name="AndroidWindowSplashScreenBackground" value="#ffffff" />
```

### `AndroidWindowSplashScreenBrandingImage`

> :warning: this setting is currently unsupported by the Android's Splash Screen compatability library. [See Google Android Issue Tracker](https://issuetracker.google.com/issues/194301890)

<!--
The splash screen's branding image.

**Supported Platforms:**

> - Android

Reviewer: The warning says "this setting is currently unsupported by the Android's Splash Screen compatibility library.", but I don't think people will know what that means, so maybe we should add here "Android 12+"?
Or edit the warning to include that the compatibility library is used on Android <= 11?

Author: For the target release and what will be supported in 11.0.0, I updated the message as such:

> > :warning: this setting is currently unsupported. The core-splashscreen compatibility library, that Cordova Android relies on for providing backwards support, has not implemented this functionality. [See Google Android Issue Tracker](https://issuetracker.google.com/issues/194301890)

How is this?

**Supported Platforms:**

- Android
- iOS

**Data Type:** `Number`, in milliseconds

**Default Value:** `true`

- Android

>   `-1`: The splash screen will automatticly hide when the `onPageFinished` has been triggered.

Reviewer 2: ```suggestion
  `-1`: The splash screen will automatically hide when the `onPageFinished` has been triggered.
```

- iOS

>   `3000`: The splash screen will automatticly hide in 3 seconds.

Reviewer 2: ```suggestion
  `3000`: The splash screen will automatically hide in 3 seconds.
```

<preference name="SplashScreenDelay" value="3000" />
```

### `AndroidPostSplashScreenTheme`

Sets the post-theme of the activity after splash screen hides.

_Note:_ This setting is available but use at your own risk.

**Supported Platforms:**

- Android

**Data Type:** `String`

**Default Value:** `@style/Theme.AppCompat.NoActionBar`

**Usage Example:**

```xml
<preference name="AndroidPostSplashScreenTheme" value="@style/Theme.AppCompat.NoActionBar"/>
```

### `AndroidWindowSplashScreenAnimatedIcon`

> The splash screen image. This preference is used for both animated and non-animated icons. The current acceptable asset files can either be an XML Vector Drwable or PNG.

Reviewer 2: ```suggestion
The splash screen image. This preference is used for both animated and non-animated icons. The current acceptable asset files can either be a XML Vector Drawable or PNG.
```

**Supported Platforms:**

- Android

**Data Type:** `String`, file path to the asset file

**Default Value:** If not supplied, the Cordova's default asset is used.

**Usage Example:**

```xml
<preference name="AndroidWindowSplashScreenAnimatedIcon" value="res/screen/android/splashscreen.xml" />
```

### `AndroidWindowSplashScreenAnimationDuration`

> Defines the duration of the icon's animation if an Animated Vector Drwable is supplied as the splash screen image.

Reviewer 2: ```suggestion
Defines the duration of the icon's animation if an Animated Vector Drawable is supplied as the splash screen image.
```
